Monika After Story/game/event-handler.rpy

```diff
@@ -2033,6 +2036,10 @@ label prompt_menu:
         if cb_label is not None:
             call expression cb_label
 
+        # allows our sub labels to utilize return keys like "love"
+        if _return is not None and "love" in _return:
```

Comment:
this flow is from idle mode, do you really anticipate using a "love" return in an idle callback?

Reply:
There is one used in an idle callback, this had to be added because of it.

Reply:
There are actually 2 returns of love in idle in this very PR, so yeah, I anticipate it.